plugin/v2/settable_test.go

```diff
@@ -89,3 +92,49 @@ func TestUpdateConfigEnv(t *testing.T) {
 		}
 	}
 }
+
+func newMount(name, source string) types.PluginMount {
```

Comment:
newMount is not returning a pointer, and it's not even needed as far as i can tell.